pyproject.toml

```diff
@@ -91,6 +91,7 @@ doc = [
   "sphinx_autosummary_accessors", # doc layout
   "sphinx",                       # build docks
   "sphinx_design",                # doc layout
+  "sphinx-autoapi",               # auto api documentation
```

Comment:
Did we not try and discard this option?

Reply:
we did, seems like I forgot to remove it. my bad